internal/service/databaseuser/model_database_user_test.go

```diff
@@ -364,3 +365,69 @@ func getDatabaseUserModel(roles, labels, scopes basetypes.SetValue, password typ
 		Scopes:           scopes,
 	}
 }
+
+func TestSplitDatabaseUserImportID(t *testing.T) {
+	const invalidDefaultString = "import format error: to import a Database User, use the format {project_id}-{username}-{auth_database_name} OR {project_id}/{username}/{auth_database_name}"
```

Comment:
can we export this error message to a common const and re-use here?

Reply:
Thanks! Changed in c866b36de8b4db820543ed6a2f791617403e361c